Mathlib/Data/Ordmap/Ordnode.lean

```diff
@@ -1285,24 +1330,28 @@ def inter : Ordnode α → Ordnode α → Ordnode α
 /-- O(n * log n). Build a set from a list, preferring elements that appear earlier in the list
 in the case of equivalent elements.
 
-    of_list [1, 2, 3] = {1, 2, 3}
-    of_list [2, 1, 1, 3] = {1, 2, 3}
+    ofList [1, 2, 3] = {1, 2, 3}
+    ofList [2, 1, 1, 3] = {1, 2, 3}
 
 Using a preorder on `ℕ × ℕ` that only compares the first coordinate:
 
-    of_list [(1, 1), (0, 1), (1, 2)] = {(0, 1), (1, 1)} -/
+    ofList [(1, 1), (0, 1), (1, 2)] = {(0, 1), (1, 1)} -/
 def ofList (l : List α) : Ordnode α :=
   l.foldr insert nil
 #align ordnode.of_list Ordnode.ofList
 
 /-- O(n * log n). Adaptively chooses between the linear and log-linear algorithm depending
   on whether the input list is already sorted.
 
-    of_list' [1, 2, 3] = {1, 2, 3}
-    of_list' [2, 1, 1, 3] = {1, 2, 3} -/
+    ofList' [1, 2, 3] = {1, 2, 3}
+    ofList' [2, 1, 1, 3] = {1, 2, 3} -/
 def ofList' : List α → Ordnode α
   | [] => nil
-  | x :: xs => if List.Chain (fun a b => ¬b ≤ a) x xs then ofAscList (x :: xs) else ofList (x :: xs)
+  | x :: xs => ofList (x :: xs)
```

Comment:
[List.decidableChain'](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/List/Defs.html#List.decidableChain') exists (do you have it imported)? Please revert to original definition

Reply:
It must have been a compiler bug or some such; now works.